docs/explanation/parts.rst

```diff
@@ -174,6 +176,10 @@ can use the :ref:`after` property to define this relationship. This property
 specifies a list containing the names of parts that it will be built after.
 The parts in the list will be *built and staged* before the part is built.
 
+By default, Craft Parts uses the defined build order to determine which
+parts can be built in parallel. This can be disabled by setting the
+:ref:`disable_parallel` property to ``True``.
+
```

Comment:
Part building is always serialized, no parts are built in parallel. `disable_parallel` is used to disable parallel builds inside a part, but it's currently not implemented. I think we can remove this paragraph for now.